src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,512 @@
+| MolarActivityMeasTime         | RECOMMENDED. Time to which molar radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                             |
+| InfusionSpeed                 | RECOMMENDED. If given, infusion speed.                                                                                                                                                                                                                                               |
+| InfusionSpeedUnit             | RECOMMENDED. Unit of infusion speed (for example, "mL/s").                                                                                                                                                                                                                           |
+| InjectedVolume                | RECOMMENDED. Injected volume of the radiotracer.                                                                                                                                                                                                                                     |
+| InjectedVolumeUnit            | RECOMMENDED. Unit of the injected volume of the radiotracer (for example, "mL").                                                                                                                                                                                                     |
+| Purity                        | RECOMMENDED. Purity of the radiolabeled compound (for example, 90).                                                                                                                                                                                                                  |
+| PurityUnit                    | RECOMMENDED. Unit of the radiochemical purity (for example, "percent").                                                                                                                                                                                                              |
+
+
+#### Time
+
+This section is mandatory and contains timing information about the imaging experiment.
+
+| Field name      | Definition                                                                                                                                                                          |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TimeZero        | REQUIRED. Time zero to which all scan and/or blood measurements have been adjusted to, in the unit “hh:mm:ss”. This should be equal to `InjectionStart` or `ScanStart`.             |
```

Comment:
Thanks @KrisThielemans, we looked at the common principles and there it is actually not clear if a time zone should be given. So @effigies and @Remi-Gau is the time zone not given in order to not reveal the location/identity?

Reply:
Well the date section in common principles does allow to indicate UTC offsets.

```
Date time information MUST be expressed in the following format YYYY-MM-DDThh:mm:ss[.000000][Z] (year, month, day, hour (24h), minute, second, optional fractional seconds, and optional UTC time indicator). This is almost equivalent to the RFC3339 "date-time" format, with the exception that UTC indicator Z is optional and non-zero UTC offsets are not indicated. If Z is not indicated, time zone is always assumed to be the local time of the dataset viewer. No specific precision is required for fractional seconds, but the precision SHOULD be consistent across the dataset.
```

Also makes me realize that the time stamp format is already mentioned in the common principle so I am not sure it is necessary to repeat it here.

Reply:
Hi @KrisThielemans, just to clarify so do you mean by this that we should not call it TimeZero? I am a bit unsure what you mean right above.

Reply:
I was wondering if BIDS needs a `DateTimeZero`. @Remi-Gau will be able to confirm I guess. (I can see why merging data and time in one keyword is good: what happens when the time zone "crosses" the date?)

Reply:
Ah, got it. Yes, let's hear @Remi-Gau @effigies @sappelhoff and @tsalo regarding this.

Reply:
Hum...

I am not sure we should merge those 2 things.

I am realizing that the `scanDate` would fit better in a `scans.tsv´ unless there is a good reason not to have it there for PET.
https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#scans-file

Reply:
yes, `scans.tsv` could be used to add the acquisition time (time of the first sample of whatever recording the scan is) for each individual scan, what would be the benefit of this field over the "scans.tsv" files?

Reply:
Seeing this [comment](https://github.com/bids-standard/bids-specification/pull/519/files#r512084391) in the BEP012 made me realize that instead of `TimeZero` we could potentially reuse `StartTime` that is already used for the `physio` part.

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/06-physiological-and-other-continuous-recordings.html

It is defined as such in physio: 

> Start time in seconds in relation to the start of acquisition of the first data sample in the corresponding neural dataset (negative values are allowed).

So i am not sure that we can just reuse the same definition in this case. 

I like the idea of having fewer "keywords" that we can consistently reuse across the specs IF their definitions are "close enough". Otherwise it might be confusing for the users if the same words means entirely different things in PET and Physio for example. 

Plus it might lead to some headaches later on. 

So if anyone feels there is potential for some conflicts later on in trying to shoe horn those PET concepts into the terminology / namespace we already have in the specs, then it might be better to sitck with TimeZero.

Reply:
Dear @effigies @Remi-Gau @sappelhoff @tsalo  and @mnoergaard, this is also definitely something we need to talk about tomorrow.